core/tools/mavp2p/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+LOCAL_BINARY_PATH="/usr/bin/mavp2p"
+VERSION="v1.1.1"
+GITHUB_REMOTE="https://github.com/bluenviron/mavp2p"
+
+# By default we install armv6
+REMOTE_BINARY_URL="${GITHUB_REMOTE}/releases/download/${VERSION}/mavp2p_v1.1.1_linux_armv7.tar.gz"
+if [[ "$(uname -m)" == "x86_64"* ]]; then
+  REMOTE_BINARY_URL="${GITHUB_REMOTE}/releases/download/${VERSION}/mavp2p_v1.1.1_linux_amd64.tar.gz"
+fi
+
+ARTIFACT_NAME="mavp2p"
+COMPRESS_FILE="$ARTIFACT_NAME.tar.gz"
```

Comment:
I'd call it COMPRESSED_FILE/ TAR_FILE or similar. `COMPRESS` sounded weird when first read it